code/modules/urist/items/uristmed.dm

```diff
@@ -39,3 +39,174 @@ Space for all Urist-done, non-pill medical items. Please keep it tidy, as usual.
 		new /obj/item/weapon/reagent_containers/syringe/antitoxin( src )
 		new /obj/item/device/healthanalyzer( src )
 		return
+
+//Rags from torn clothes, a quick and, literally, dirty way to stop the bleeding... eventually.
+
+/obj/item/weapon/clothesrag
+	name = "rag"
+	desc = "The sad remains of a piece of clothing. Could be used to press wounds to stop them from bleeding, but it might take some time and is of dubious cleanliness."
+	w_class = 1
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "rag"
+	var/bloodvolume = 0 //how much blood has the rag soaked up
+	var/bloodcapacity = 5 //how many times the rag can soak up blood
+	var/soaked = 0 //if the rag has reached its capacity
+
+/obj/item/weapon/clothesrag/proc/checkvolume()
+	if(src.bloodvolume >= src.bloodcapacity)
+		src.soaked = 1
+	else
+		src.soaked = 0
+	if(soaked)
+		usr << "The rag has reached its capacity!"
+		src.name = "bloody rag"
+		src.desc = "The sad remains of a piece of clothing. It has soaked up too much blood to be useful for stopping bleeding, better wash it!"
+		icon_state = "rag"
+	else
+		src.name = "rag"
+		src.desc = "The sad remains of a piece of clothing. Could be used to press wounds to stop them from bleeding, but it might take some time and is of dubious cleanliness."
+		icon_state = "rag"
+
+/obj/item/weapon/clothesrag/clean_blood()
+	..()
+	src.bloodvolume = 0
+	src.checkvolume()
+
+/obj/item/weapon/clothesrag/attack(mob/living/carbon/M as mob, mob/user as mob) //copypasta ahoy!
+
+	if(soaked)
+		user << "<span class='warning'> \The [src] is too bloody!</span>"
+		return 1
+
+	if (!istype(M))
+		user << "<span class='warning'> \The [src] cannot be applied to [M]!</span>"
+		return 1
+
+	if ( ! (istype(user, /mob/living/carbon/human) || \
+			istype(user, /mob/living/silicon) || \
+			istype(user, /mob/living/carbon/monkey)) )
+		user << "<span class='warning'> You don't have the dexterity to do this!</span>"
+		return 1
+
+	if (istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		var/datum/organ/external/affecting = H.get_organ(user.zone_sel.selecting)
+
+		if(affecting.display_name == "head")
+			if(H.head && istype(H.head,/obj/item/clothing/head/helmet/space))
+				user << "<span class='warning'> You can't apply [src] through [H.head]!</span>"
+				return 1
+		else
+			if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/space))
+				user << "<span class='warning'> You can't apply [src] through [H.wear_suit]!</span>"
+				return 1
+
+		if(affecting.status & ORGAN_ROBOT)
+			user << "<span class='warning'> This isn't useful at all on a robotic limb..</span>"
+			return 1
+
+		H.UpdateDamageIcon()
+
+		if(affecting.open == 0)
+			if(!affecting.bandage())
```

Comment:
Wounds are getting bandaged at this point, so prob check down the line does nothing.